layerstack/tests/test_layer_creation.py

```diff
@@ -40,3 +40,10 @@ def test_layer_base():
     _layer_dir = Layer.create('Test Layer Base',layer_library_dir)
     # should be able to run the layer as-is
     subprocess.check_call(['python', str(layer_library_dir / 'test_layer_base' / 'layer.py'), 'dummy_arg'])
+
+
+# *** test issue 19: Layer.__init__ supports model keyword argument (passes newly added self._model)
+def test_model_kwarg_layer_init():
+    _layer_dir = Layer.create('Test Model Layer Base',layer_library_dir, ModelLayerBase)
+    # should be able to run the layer as-is
+    subprocess.check_call(['python', str(layer_library_dir / 'test_model_layer_base' / 'layer.py'), 'dummy_arg'])
```

Comment:
This is not a full test of the functionality that you are providing.

I would 
- [ ] Create a new test_model_layer_base.py file in layerstack/tests
- [ ] There, define some really simple "model class"
- [ ] Add a test_model_layer_base layer to layerstack/tests/layerlibrary
- [ ] Edit the resulting layer.py file to have an arg or two and a kwarg or two; AND make it so that at least one of them can be influenced by a model (of the "model class" type you defined in test_model_layer_base.py)
- [ ] Use that behavior to test that your functionality is used when you directly load the Layer from disk
- [ ] Use that behavior to test that your functionality is used on the command line

Reply:
It is not straightforward with the current design to pass a model argument into args and kwargs when running a layer from the command line. It would be nice to have an option on the command line to list a layer's args and kwargs given a particular model.